bower.json

```diff
@@ -8,13 +8,13 @@
   ],
   "dependencies": {
     "fxos-mvc": "git@github.com:fxos/mvc.git",
-    "gaia-theme": "gaia-components/gaia-theme",
-    "gaia-header" : "gaia-components/gaia-header",
-    "gaia-list": "gaia-components/gaia-list",
-    "gaia-button": "gaia-components/gaia-button",
-    "gaia-fonts": "gaia-components/gaia-fonts",
-    "gaia-dialog": "gaia-components/gaia-dialog",
-    "gaia-tabs": "gaia-components/gaia-tabs",
-    "gaia-sub-header": "gaia-components/gaia-sub-header"
+    "gaia-theme": "gaia-components/gaia-theme#0.4.5",
```

Comment:
I figure since this app can pull all it's dependencies when it's being built within lightsaber, might as well lock down the versions completely.